extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/StoreType.java

```diff
@@ -14,7 +15,11 @@
     }
 
     public boolean isDbStore() {
-        return RAM != this;
+        return RAM != this && CUSTOM != this;
```

Comment:
```suggestion
        return JDBC_CMT == this || JDBC_TX == this;
```
nitpick: we can now change it to reference the db stores types.

Reply:
changed